jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java

```diff
@@ -258,12 +258,6 @@ protected void setEditorListener(final Consumer<FilterSettings> editorListener)
         this.editorListener = editorListener;
     }
 
-    @Override
-    public void onHide(Event e) {
-        super.onHide(e);
-        filtersControlPanel.clear();
```

Comment:
@nmirasch seems like filters are empty when opening the popup, unless I'm missing some edge case, wdyt?

Reply:
yep, I think it's not needed that clear